pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+
+
+> Before submitting the pull request, you must agree with the following statements by checking the boxes with a 'x'.
```

Comment:
"by checking **all** the boxes" maybe? (to be sure both need to be checked)

Reply:
Done, replaced "the boxes" with "both boxes".